sdk/mobile-center-updates/src/main/java/com/microsoft/azure/mobile/updates/Updates.java

```diff
@@ -602,7 +606,33 @@ private synchronized void getLatestReleaseDetails(@NonNull String updateToken) {
         Map<String, String> headers = new HashMap<>();
         headers.put(HEADER_API_TOKEN, updateToken);
         final Object releaseCallId = mCheckReleaseCallId = new Object();
-        mCheckReleaseApiCall = httpClient.callAsync(url, METHOD_GET, headers, null, new ServiceCallback() {
+        mCheckReleaseApiCall = httpClient.callAsync(url, METHOD_GET, headers, new HttpClient.CallTemplate() {
+
+            @Override
+            public String buildRequestBody() throws JSONException {
+
+                /* This method is getting called. */
```

Comment:
Actually this method is not called for a get.
I would either remove this comment or change it to something like "No request body as its a get call"